e2e-tests/popup/review-flow/review-flow.spec.ts

```diff
@@ -0,0 +1,331 @@
+
+      popupExpect(
+        popupPage.getByRole('heading', {
+          name: "We'd love to hear more from you"
+        })
+      ).toBeVisible();
+
+      popupExpect(
+        popupPage.getByRole('button', { name: 'Get in touch' })
+      ).toBeVisible();
+
+      const [newPage] = await Promise.all([
+        context.waitForEvent('page'),
+        page.getByRole('button', { name: 'Get in touch' }).click()
+      ]);
+
+      expect(newPage.url()).toBe('https://t.me/CSPRhub/4689');
+    }
+  );
+
+  popup(
+    'negative review flow - go to home page',
+    async ({ popupPage, unlockVault }) => {
+      await unlockVault();
+
+      await popupPage.route(URLS.rpc, route =>
+        route.fulfill(RPC_RESPONSE.success)
+      );
+
+      await new Promise(r => setTimeout(r, 5000));
+
+      await popupPage.getByText('Send').click();
+
+      await popupExpect(
+        popupPage.getByRole('heading', { name: 'Select token and account' })
+      ).toBeVisible();
+
+      await popupPage.getByRole('button', { name: 'Next' }).click();
+
+      await popupExpect(
+        popupPage.getByRole('heading', { name: 'Specify recipient' })
+      ).toBeVisible();
+
+      await popupExpect(
+        popupPage.getByRole('button', { name: 'Next' })
+      ).toBeDisabled();
+
+      await popupPage
+        .getByPlaceholder('Public key or name', { exact: true })
+        .fill(DEFAULT_SECOND_ACCOUNT.publicKey);
+
+      await popupExpect(
+        popupPage.getByText(DEFAULT_SECOND_ACCOUNT.mediumTruncatedPublicKey, {
+          exact: true
+        })
+      ).toBeVisible();
+
+      await popupPage
+        .getByText(DEFAULT_SECOND_ACCOUNT.mediumTruncatedPublicKey, {
+          exact: true
+        })
+        .click();
+
+      await popupPage.getByRole('button', { name: 'Next' }).click();
+
+      await popupExpect(
+        popupPage.getByRole('heading', { name: 'Enter amount' })
+      ).toBeVisible();
+
+      await popupPage.getByRole('button', { name: 'Next' }).click();
+
+      await popupExpect(
+        popupPage.getByRole('heading', { name: 'Confirm sending' })
+      ).toBeVisible();
+
+      await popupExpect(
+        popupPage.getByText(DEFAULT_SECOND_ACCOUNT.publicKey)
+      ).toBeVisible();
+
+      await popupExpect(
+        popupPage.getByRole('button', { name: 'Confirm send' })
+      ).toBeDisabled();
+
+      // Scroll to the bottom
+      await popupPage.evaluate(() => {
+        const container = document.querySelector('#ms-container');
+
+        container?.scrollTo(0, 1000);
+      });
+
+      await popupExpect(
+        popupPage.getByRole('button', { name: 'Confirm send' })
+      ).not.toBeDisabled();
+
+      await popupPage.getByRole('button', { name: 'Confirm send' }).click();
+
+      await popupExpect(
+        popupPage.getByRole('heading', { name: 'You submitted a transaction' })
+      ).toBeVisible();
+
+      await popupPage.getByRole('button', { name: 'Done' }).click();
```

Comment:
this code is used 3 times in this file. Maybe try to reuse it somehow and not duplicate